lib/imgix/path.rb

```diff
@@ -75,10 +75,14 @@ def method_missing(method, *args, &block)
 
     ALIASES.each do |from, to|
       define_method from do |*args|
+        warn "Warning `Path.#{from}' has been deprecated and " \
+             "will be removed in the next major version.\n"
```

Comment:
I like this message as is! I wonder if it's worth calling out to the deprecation of `ALIASES` as well. It's a weak opinion but was curious what you think:

```suggestion
        warn "Warning `Path.#{from}' (and all parameter `ALIASES`) has been deprecated and " \
             "will be removed in the next major version.\n"
```

Reply:
Yeah! Let's!